flink-autoscaler/src/main/java/org/apache/flink/autoscaler/JobAutoScalerImpl.java

```diff
@@ -171,7 +171,7 @@ private void runScalingLogic(Context ctx, AutoscalerFlinkMetrics autoscalerMetri
             stateStore.storeScalingTracking(ctx, scalingTracking);
         }
 
-        if (collectedMetrics.getMetricHistory().isEmpty()) {
+        if (collectedMetrics.getMetricHistory().size() < 2) {
```

Comment:
Would you mind adding a simple comment to explain why we call the scaling logic when metric history size >= 2?

Reply:
will do!